ursula_cli/shell.py

```diff
@@ -192,10 +192,13 @@ def _run_heat(args, hot):
         LOG.error("You must have python-heatclient in your python path")
         raise Exception(e)
 
+    tenant_var = "OS_TENANT_NAME"
+    if 'OS_TENANT_NAME' not in os.environ:
+        tenant_var = 'OS_PROJECT_NAME'
     CREDS = {
         'username': os.environ['OS_USERNAME'],
         'password': os.environ['OS_PASSWORD'],
-        'tenant_name': os.environ['OS_TENANT_NAME'],
+        'tenant_name': os.environ[tenant_var],
```

Comment:
The problem is with os.environ['PROJECT_NAME'], if that's not defined (even if TENANT_NAME is) python will throw a key error. So we could do this:

'tenant_name': os.environ.get('OS_TENANT_NAME', os.environ.get('OS_PROJECT_NAME')

However if both are undefined, you won't get your key error, you'll just get a 401 as no tenant was passed.

What needs to happen is all of those os.environ['var_names'] need to be checked with a .get and throw an error if undefined. I can do that if need be in this PR, i was thinking this was needed as a quick fix.